mm2src/coins/lp_coins.rs

```diff
@@ -3511,3 +3520,27 @@ where
         b.block_height.cmp(&a.block_height)
     }
 }
+
+#[derive(Display, Debug)]
+pub enum RpcCommonError {
+    FindClientError(String),
+    WrongRpcClient,
+}
+
+#[derive(Debug)]
+pub enum RpcClientEnum {
```

Comment:
Actually thank you for this hint!
I removed enums due to adding types in trait.
So I can just implement this. it's like you suggested below.
```rust
impl RpcCommonOps for TendermintCoin {
    type RpcClient = HttpClient;
    type Error = TendermintCoinRpcError;

    async fn get_live_client(&self) -> Result<Self::RpcClient, Self::Error> {
``` 

please note, that we use `get_live_client` function like one line in `rpc_client` method from trait `TendermintCommons`. 
But I can not remove `rpc_client` from trait and use just  `get_live_client` instead, bcz we need this function regarding `trait CoinCapabilities: TendermintCommons + CoinWithTxHistoryV2 + MmCoin + MarketCoinOps {}`.
This explanation is just in case there is a question whether `rpc_client` should be removed.